_posts/0000-01-02-kmsargen.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
#This would be an awesome change- to make the text an H1 using markdown... another language to learn.#